src/View/Helper/AbstractTranslatorHelper.php

```diff
@@ -28,7 +28,7 @@
      *
      * @var string
      */
-    protected $translatorTextDomain = 'default';
+    protected $translatorTextDomain = null;
```

Comment:
This is a slight BC Break - needs to be documented, plus I really wouldn't do it, unless strictly needed